debug_toolbar/panels/templates/panel.py

```diff
@@ -121,7 +121,7 @@ def _store_template_info(self, sender, **kwargs):
             return
 
         context_list = []
-        for context_layer in context.dicts:
+        for context_layer in getattr(context, 'dicts', []):
```

Comment:
Indeed. Respectively `getattr(context, 'dicts', [context])`, otherwise the AttributeError will still happen.